front/gatsby/src/pages/Validators.tsx

```diff
@@ -4,35 +4,38 @@
 
 import { useQuery } from '@apollo/react-hooks';
 import { RouteComponentProps } from '@reach/router';
+import { Spinner } from '@substrate/design-system';
 import React, { useEffect, useState } from 'react';
 
 import { LATEST_SESSION_QUERY } from '../util/graphql';
-import { CurrentElectedList } from './CurrentElected';
+import { CurrentElectedList } from '../components/CurrentElected';
 
 type Props = RouteComponentProps;
 
 const Validators = (_props: Props): React.ReactElement => {
-  const { data } = useQuery(LATEST_SESSION_QUERY, {
-    pollInterval: 10000,
-  });
+  
   const [sessionIndex, setSessionIndex] = useState<number>();
 
   useEffect(() => {
+    const { data } = useQuery(LATEST_SESSION_QUERY, {
+      pollInterval: 10000,
+    });
```

Comment:
Nice! so this solves the apollo error on gatsby build? Do you have an idea why?

Reply:
it's absolutley messed up. This makes it build, but it violates React rule of hooks in `gatsby develop`, i.e. hook cannot be called within a hook.